### PR TITLE
Class02 hw3 kihoon

### PR DESCRIPTION
**Pull request by KORIDO1234**
Class02 hw3 kihoon

### CODE REVIEW

File: class02/homework/kihoon/hw1/led.py
Comment (JerryJLee) [inline class02/homework/kihoon/hw1/led.py:1]: 해당 파일은 첫번째 과제용이죠. 불필요하므로 커밋에 포함시키지 않도록 합시다!

File: class02/homework/kihoon/hw3/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+./splitted_dataset/train/good: 24
+```
+
+## Training 결과
+|Classification model|Accuracy|FPS|Training time|Batch size|Learning rate|Other hyper-prams|
+|----|----|----|----|----|----|----|
+|EfficientNet-V2-S| |
+|EfficientNet-B0|
+|DeiT-Tiny| 
+|MobileNet-V3-large-1x|1.0|1.06|0:00:06.846977|64|0.0058|----|----|
```
Comment (JerryJLee) [inline class02/homework/kihoon/hw3/README.md:21]: 모델 하나만 성공 했나요?
